docs/source/index.rst

```diff
@@ -11,18 +11,20 @@ Welcome to Hyperledger Indy Node's documentation!
    :maxdepth: 1
 
    transactions.md
-   requests.md
+   requests-new.md
```

Comment:
The valid request description is in `requests.md`. `requests-new.md` is not implemented yet, so I think it doesn't make sense to include it into index.